### ISSUE TITLE
Add pycurl to one of dependencies in README.md

### ISSUE THREAD

**Issue by elogada** [nahamsec/JSParser#36]
I tried installing the JSParser straight out of the box as per README.md's requirements. Although I am met with problems, I still managed to fix them.

The (main) problem was that in the box that I was using, pycurl had not been included yet. Whether this is a side-effect of Python2.7's deprecation in 2020 is out of my reach, however, simply installing pycurl managed to do the job.

The syntax for this hotfix is

`````
pip install pycurl
`````

In the event that one is met with more errors, a significant help would be to install libgnutls* packages from one's repository. In that case, the fix would be dependent on your default-installation _and_ what you can get from your repo.

Tl;dr
Add `pycurl` as a dependency in the Readme file.
